susemanager-utils/susemanager-sls/salt/virt/create-vm.sls

```diff
@@ -12,6 +12,24 @@ initrd_cached:
     - makedirs: True
 {%- endif %}
 
+{%- if 'interfaces' in pillar %}
+nets-{{ pillar['name'] }}:
+  virt_utils.network_running:
+    - networks:
+    {%- for nic in pillar['interfaces'] %}
+      - {{ nic['source'] }}
+    {%- endfor %}
+{%- endif %}
+
+pools-{{ pillar['name'] }}:
```

Comment:
Shouldn't we check `{%- if 'disks' in pillar %}`, in the same way we do for "interfaces", to avoid breaking the SLS in case pillar['disks'] does not exist?

Reply:
I'll add it, even if it may sound silly to create a VM without any disk